### ISSUE TITLE
❌ Software Removal | Keka

### ISSUE THREAD

**Issue by KurnTu59** [privacytools/privacytools.io#2126]
## Description
I suggest removing Keka.

## Why I am making the suggestion
Keka was once open source.
But now it's closed source.

Transparency and mass surveillance are important for cryptographic software.
I suggest removal.

## My connection with the software
I have nothing to do with the Keka developers.

- [✅] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by samuel-lucas6** [comment]
I agree that it makes sense to remove Keka if you want to clean up the Worth Mentioning section as mentioned [here](https://github.com/privacytools/privacytools.io/pull/2100). However, I'm biased since my application is on this page.

**Comment by lynn-stephenson** [comment]
I was already looking into removing Keka. On the research of the program's history, and trying to come up with a decent conclusion whether or not the pogram should be trusted;

I determined that the author of the pogram closed it because somebody else had taken his code and submitted to the AppStore for the MacOS (if I recall correctly, it's been a few months and I didn't keep notes).

Which isn't really a good reason for close-sourcing your program, I believe. And I don't think an archiving tool with "meh" encryption built-in really fits as a cryptographic tool with security in mind.

For Keka having encryption support _seems_ to be more about having it as an additional feature, rather than incorporating security as a process.

I can do additional research, but Keka should _probably_ be removed on the basis of not fitting into this section, and being closed source (and for silly reasons).

Even if it is a "good" program, it needs moved to a different section.

**Comment by lynn-stephenson** [comment]
7Zip also does not belong in this section, so I will probably get both of these removed.

And it isn't really a privacy tool. Just a simple archiving tool.

**Comment by lynn-stephenson** [comment]
7Zip also does not belong in this section, so I will probably get both of these removed.

And it isn't really a privacy tool. Just a simple archiving tool.

**Comment by samuel-lucas6** [comment]
7-Zip does make it easy to password protect a ZIP, but I agree that perhaps it doesn't belong in this section or maybe it should be moved to Worth Mentioning. @lynn-stephenson Where would you move it to?

I think I'd replace 7-Zip's spot with Cryptomator for the file encryption recommendation. Is LUKS only in Worth Mentioning because of how difficult it is to use the tool compared to VeraCrypt? LUKS should be recommended for disk encryption on Linux really since it's built into loads of distros and VeraCrypt doesn't support system drive encryption on Linux.

**Comment by KonoromiHimaries** [comment]
> but I agree that perhaps it doesn't belong in this section or maybe it should be moved to Worth Mentioning

May be replaced with https://mhogomchungu.github.io/sirikali/
